### DISCUSSION TITLE
How to Specify Multiple Lists in PnP - Search Filters

### DISCUSSION THREAD

**Discussion by tatsutanaka** [microsoft-search/pnp-modern-search#3993] [Q&A]
I have a question regarding the PnP Modern Search Web Parts.

If I want to specify multiple lists in "PnP - Search Filters" and limit the search scope to items within those specified lists, how should I configure the web part settings?

**Comment by wobba** [comment]
You cannot target a list via filters. You need to figure out a different setup with maybe two pages, one per list. And then have hyperlinks on pages to allow navigation between them.
